modules/sandboxed-containers-installing-operator-cli.adoc

```diff
@@ -3,5 +3,89 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-installing-operator-cli_{context}"]
+= Installing the {sandboxed-containers-operator} using the CLI
 
-= Installing the Sandboxed Containers Operator using the CLI
+You can install the {sandboxed-containers-operator} using the {product-title} CLI.
+
+.Prerequisites
+
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have subscribed to the {sandboxed-containers-first} catalog.
++
+[NOTE]
+====
+Subscribing to the {sandboxed-containers-first} catalog provides `openshift-sandboxed-containers` namespace access to the {sandboxed-containers-operator}.
+====
+
+.Procedure
+
+. Create a YAML file that contains the following manifest:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-sandboxed-containers
+
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-sandboxed-containers-kataconfig-group
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  targetNamespaces:
+    - openshift-sandboxed-containers
```

Comment:
should be openshift-sandboxed-containers-operator

Reply:
Meta-question: wouldn't it make sense to provide the file on-line and to recommend an `oc apply -f http://<something>` ? That way, if we make minute changes in it, we don't need to update the doc every time.

Reply:
you are supplying the content of the file here anyways, if the link does not work, then the user won't be able to install in that case.

Reply:
I know that I personally find it inconvenient to have to create a file for such mundane operations. Maybe because my network is slow/unreliable enough that copy-pasting large chunks of text to a remote machine is known to sometime fail in mysterious ways. BTW, I was not suggesting we _only_ provide the remote version, but that we do offer one.

Reply:
@sagidlow targetNamespaces needs to be "openshift-sandboxed-containers-operator"